src/components/rating/rating.ts

```diff
@@ -126,6 +131,10 @@ export default class IgcRatingComponent extends SizableMixin(
   @property({ type: Boolean, reflect: true })
   public readonly = false;
 
+  /** Toggles single selection visual mode. */
+  @property({ type: Boolean })
```

Comment:
I suppose this should be a reflected property 
```suggestion
  @property({ type: Boolean, reflect: true })
```

Reply:
This is not needed for styling, so no.